public_html/index.php

```diff
@@ -348,9 +348,9 @@
       <div class="card h-100 shadow-sm">
         <div class="card-body d-flex align-items-center">
           <div>
-            <h5 class="card-title">Cloud-ready</h5>
-            <p class="card-text">Pipelines are tested on AWS after every release.
-              You can even browse results live on the website and use outputs for your own benchmarking.
+            <h5 class="card-title">Robust</h5>
```

Comment:
Why remove `Cloud-ready`?

Reply:
I would keep "Cloud-ready" as we use containers for everything

Reply:
But we do not test the results on cloud. We think it will work on the cloud but we do not have tests to show for it. And at this time we cannot browse our results externally, at least based on what I know.

Reply:
All NF pipelines are supposed to work on the cloud? We can treat our farm as cloud.

Reply:
Some pipelines are properly tested on GitHub CI, which amounts to a cloud. There are two issues, I think:

- I think it's only TreeVal that's currently incompatible because it needs a lot of data from Lustre.
- We hit some problems with Busco, but I think it's essentially because we wanted to force running it on a specific test dataset. We could just skip the parameter and the pipeline should happily run, downloading the dataset live from the Busco website

Reply:
If we are keeping `cloud-ready`, I will not change the logo. Otherwise I need to change the `aws` logo too.

Reply:
I feel one of the features of "cloud ready" is the ability of pipelines to create instances as needed within the given parameters while talking to all the components successfully. We have not tested any pipeline on a full dataset on the cloud (even GitHub). 

But if there is general consensus, I can put it back in.

Reply:
I am adding back cloud ready. Should we change the image to a cloud or leave it as `aws`, seems like free advertising.

Reply:
That just represents the cloud, make less change as possible.

Reply:
Fixed